### PR TITLE
Add landscape docs

### PR DESCRIPTION
**Pull request by nottrobin**
Build landscape docs, link to them fromm the homepage, setup redirect
for the front page.

QA
--

``` bash
./rebuild-docs # Build docs including landscape
./run          # Run server
```

Go to <http://127.0.0.1:8007> and browse to landscape docs in all their
glory.

### CODE REVIEW

**Comment by nottrobin** [comment]
[make Landscape docs live](https://trello.com/c/jP08EIX3/2958-make-landscape-docs-live)

File: redirects.yaml [hunk 1]
```diff
@@ -14,7 +13,13 @@ maas(/|/en/?|/2.1/?)?: /maas/2.1/en/
 # Pages redirects:
 # On 12/01/2017
 core/en/guides/build-device/assertions: /core/en/reference/assertions
+core/en/guides/build-device/assertions: /core/en/reference/assertions
```
Comment (anthonydillon) [inline redirects.yaml]: Duplication of the line above.

File: templates/index.html [hunk 2]
```diff
@@ -22,38 +22,61 @@
...
+          <thead>
+              <tr><th>Project</th><th>Description</th></tr>
+          </thead>
+          <tbody>
+              <tr>
+                  <td><a href="https://www.ubuntu.com/desktop" class="p-link--external">Official Ubuntu documentation</a></td>
+                  <td>
+                      <p>Documentation for the Ubuntu operating system.</p>
+                      <p><a href="https://XXXX" class="p-link--external">Read the official Ubuntu documentation</a></p>
+                  </td>
```
Comment (anthonydillon) [inline templates/index.html]: Should remove the desktop docs until ready.

File: redirects.yaml (hunk 1 above)
Comment (nottrobin) [inline redirects.yaml]: Fixed

File: templates/index.html (hunk 2 above)
Comment (nottrobin) [inline templates/index.html]: It is "ready", it was mean to send you to https://help.ubuntu.com/, but I forgot the URL at the time. I've added it now.

File: templates/index.html [hunk 3]
```diff
@@ -22,38 +22,61 @@
...
+      </table>
+
+      <h2>Externally hosted documentation</h2>
+      <table>
+          <thead>
+              <tr><th>Project</th><th>Description</th></tr>
+          </thead>
+          <tbody>
+              <tr>
+                  <td><a href="https://www.ubuntu.com/desktop" class="p-link--external">Official Ubuntu documentation</a></td>
```
Comment (pmahnke) [inline templates/index.html]: just link to www.ubuntu.com I think...

**Pull request merged**